.circleci/config.yml

```diff
@@ -84,8 +84,63 @@ jobs:
       - run:
           name: Cargo Test --target x86_64-pc-windows-msvc
           command: cargo test --target x86_64-pc-windows-msvc
+  utp-test:
+    description: |
+      Run uTP network simulator
+    machine:
+      image: ubuntu-2004:current
+    resource_class: xlarge
+    environment:
+      RUSTFLAGS: '-D warnings'
+      RUST_LOG: 'debug'
+    steps:
+      - checkout
+      - run:
+          name: Update package sources
+          command: sudo apt update
+      - run:
+          name: Install libssl-dev for openssl-sys
+          command: sudo apt install -y libssl-dev
+      - run:
+          name: Install libclang for rocksdb
+          command: sudo apt install clang
+      - run:
+          name: Install modprobe
+          command: sudo apt install -y kmod
+      - run:
+          name: Load iptables6 kernel modules
+          command: sudo modprobe ip6table_filter
+      - run:
+          name: Install Rust
+          command: curl https://sh.rustup.rs -sSf | sh -s -- -y
+      - run:
+          name: Build utp-testing package
+          command: cargo build -p utp-testing --release
+      - run:
+          name: Move docker artifacts
+          command: mv -t utp-testing/docker/circleci/ target/release/utp-test-app utp-testing/docker/run_endpoint.sh utp-testing/docker/setup.sh
+      - run:
+          name: Build uTP Test App container
+          command: docker build -t test-utp --no-cache utp-testing/docker/circleci/
+      - run:
+          name: Run Test App with network simulator
+          command: SCENARIO="drop-rate --delay=15ms --bandwidth=10Mbps --queue=25 --rate_to_client=0 --rate_to_server=0" docker-compose -f utp-testing/docker/docker-compose.yml up -d
```

Comment:
It seems to me like we are only running the utp transfer test under a single specific scenario? Is the idea to eventually run the same utp transfer test under multiple scenarios with various network attributes (aka different delay, bandwidth, etc)?

Reply:
It is not a problem to docker-compose multiple networks with different scenarios but I'm not sure if we will need that. 
If we can transfer a significant amount of bytes during worst case scenario possible (some drop rate, big delay, low bandwidth), I think this will be enough.